src/app/zap-templates/zcl/data-model/chip/chime-cluster.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/Chime.adoc
+Parameters: in-progress 
+Git: 0.7-fall2024-ncr-1158-g413e195c2
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+  <struct name="ChimeSoundStruct" apiMaturity="provisional">
+    <cluster code="0x0556"/>
+    <item fieldId="0" name="ChimeId" type="int8u"/>
+    <item fieldId="1" name="Name" type="char_string" length="48"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <domain name="General"/>
+    <name>Chime</name>
+    <code>0x0556</code>
+    <define>CHIME_CLUSTER</define>
+    <description>This cluster provides facilities to configure and play Chime sounds, such as those used in a doorbell.</description>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="INSTALLED_CHIME_SOUNDS" type="array" entryType="ChimeSoundStruct">InstalledChimeSounds</attribute>
+    <attribute code="0x0001" side="server" define="ACTIVE_CHIME_SOUND_ID" type="int8u" min="0" max="255" default="0" writable="true">ActiveChimeSoundId</attribute>
```

Comment:
Again, should be "ID", not "Id", but that's a spec issue.

Reply:
Is it worthwhile making a comment[ in the spec PR ](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10004)to fix all these? [I generated this from that branch]

Reply:
@aronrosenberg updated the spec & I re-generated using alchemy 👍